src/Features/CSharp/Portable/project.json

```diff
@@ -2,6 +2,8 @@
   "dependencies": {
   },
   "frameworks": {
-    ".NETPortable,Version=v4.5,Profile=Profile7": {}
+    "netstandard1.3": {
+      "imports": [ "portable-net45+win8", "dotnet" ]
```

Comment:
If you have multiple, it'll break fast up-to-date checks. @rchande is aware of the actual bug and can help you with it. The workaround is to only have one, if you can get away with it. If you really need multiple, well, @rchande might have another idea.